onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -972,38 +885,23 @@ def _where_input_wrangler(
     TorchLibOpInfo("reshape", core_ops.aten_reshape),
     TorchLibOpInfo("resolve_conj", core_ops.aten_resolve_conj),
     TorchLibOpInfo("resolve_neg", core_ops.aten_resolve_neg),
-    TorchLibOpInfo(
-        "round",
-        core_ops.aten_round,
-    )
+    TorchLibOpInfo("round", core_ops.aten_round)
     .xfail(
         variant_name="decimals_0",
         reason="The op does not support decimals yet",
         test_class_name="TestOutputConsistencyEager",
     )
-    .xfail(
```

Comment:
how come so many formatting changes?

Reply:
Unrelated changes. I will split to a separate pr.

Reply:
Done